src/comment/comment.module.ts

```diff
@@ -3,9 +3,10 @@ import { CommentController } from './comment.controller';
 import { CommentService } from './comment.service';
 import { Comment } from './entities/comment.entity';
 import { TypeOrmModule } from '@nestjs/typeorm';
+import { LoggerModule } from 'src/common/logger/logger.module';
 
 @Module({
```

Comment:
If logger is used in almost every module, we can make it a global module, that way we do not need to import it in every module.

But be careful why making any module global.